operators/vita49_psd_packetizer/CMakeLists.txt

```diff
@@ -0,0 +1,52 @@
+# SPDX-FileCopyrightText: 2024 Valley Tech Systems, Inc.
+#
+# SPDX-License-Identifier: Apache-2.0
+cmake_minimum_required(VERSION 3.20)
+project(vita49_psd_packetizer CXX)
+
+find_package(holoscan 2.5.0 REQUIRED CONFIG
+  PATHS "/opt/nvidia/holoscan" "/workspace/holoscan-sdk/install")
+
+set(CMAKE_CUDA_ARCHITECTURES "70;80;90")
+enable_language(CUDA)
+
+include(FetchContent)
+FetchContent_Declare(
+    vrtgen
+    GIT_REPOSITORY https://github.com/Geontech/vrtgen.git
+    GIT_TAG master
+)
+FetchContent_MakeAvailable(vrtgen)
+
+FetchContent_Declare(
+  MatX
+  GIT_REPOSITORY https://github.com/NVIDIA/MatX.git
+  GIT_TAG v0.9.0
+)
+FetchContent_MakeAvailable(MatX)
+
+add_library(packetlib
+    packet_sender.cpp
+    signal_data.cpp
+    context.cpp
+)
```

Comment:
Note that the LGPL `context.cpp` component is typically incompatible with other Apache-2.0 sources.